lib/ansible/modules/cloud/ovirt/ovirt_vm.py

```diff
@@ -1374,7 +1379,7 @@ def _migrate_vm(self, entity):
                 current_vm_host = hosts_service.host_service(entity.host.id).get().name
                 if vm_host != current_vm_host:
                     if not self._module.check_mode:
-                        vm_service.migrate(host=otypes.Host(name=vm_host))
+                        vm_service.migrate(host=otypes.Host(name=vm_host), force=self.param('force_migrate'))
```

Comment:
Hmm, in order to not break backward compatibility we need to set force_migrate to false if not specified:

```
vm_service.migrate(
    host=otypes.Host(name=vm_host), 
    force=self.param('force_migrate')
        if self.param('force_migrate') is not None
        else False,
)
```

Reply:
Please ignore, the param will be set to False when not specified